pkg/services/alerting/ticker.go

```diff
@@ -42,9 +42,14 @@ func NewTicker(c clock.Clock, interval time.Duration, metric *metrics.Ticker) *T
 	return t
 }
 
+func getStartTick(clk clock.Clock, interval time.Duration) time.Time {
```

Comment:
Should we either round up or round down to the nearest second?

Reply:
It rounds down to the interval. As long as the interval is 10 seconds (currently hardcoded) it will round to times of 10 seconds.
For example `2022-06-06 17:15:27.9064497` will be rounded to `2022-06-06 17:15:20`